File: 242. Valid Anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.size() != t.size())
            return false;
        vector <int> freq_s (26), freq_t (26);
        for (int i = 0; i < s.size(); i++)
        {
            freq_s[s[i] - 'a'] += 1;
            freq_t[t[i] - 'a'] += 1;
        }
        return freq_s == freq_t ? true :false;
    }
};

File: 242. Valid Anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s) != len(t):
            return False
        freq_s, freq_t = [0 for i in xrange(26)], [0 for i in xrange(26)]
        for i in xrange(len(s)):
            freq_s[ord(s[i]) - ord('a')] += 1
            freq_t[ord(t[i]) - ord('a')] += 1
        return True if freq_s == freq_t else False
